src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/Messages.java

```diff
@@ -86,21 +87,34 @@ public void error(String key, Object... args) {
     /**
      * Reports an error message to the doclet's reporter.
      *
-     * @param path a path identifying the position to be included with
-     *  the message
-     * @param key the name of a resource containing the message to be printed
+     * @param path a path identifying the position to be included with the message
+     * @param key  the name of a resource containing the message to be printed
      * @param args optional arguments to be replaced in the message.
      */
     public void error(DocTreePath path, String key, Object... args) {
         report(ERROR, path, resources.getText(key, args));
     }
 
+    /**
+     * Reports an error message to the doclet's reporter.
+     *
+     * @param fo    the file object to be associated with the message
+     * @param start the start of a range of characters to be associated with the end
+     * @param pos   the position to be associated with the end
+     * @param end   the end of a range of characters to be associated with the end
```

Comment:
"Associated with end"? I believe it should either be "error" or "message", not "end".